.github/workflows/nextjs.yml

```diff
@@ -35,9 +35,6 @@ jobs:
         id: detect-package-manager
```

Comment:
My suggestion is to have different PRs for this in the future. Cleanup workflows file and UI work can be separated into different PRs.

Reply:
Ok sure. I will change the branch name after this PR.

Reply:
Ideally you use different PR for each task

features/<descriptionOfTask>

Reply:
yes I understand. I will change the branch name after this PR. Or do you want me to change the branch name now for this PR?

Reply:
In the future is fine :)